mgmtsystem_kpi/__openerp__.py

```diff
@@ -65,6 +70,5 @@
     ],
     "demo": [],
     "test": [],
-    'installable': False,
+    'installable': True,
 }
```

Comment:
@max3903 To be honest, this module could be independent from Management System - I can see it useful for other use cases. But let's focus on the port now, and maybe consider this move later on.